Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell.swift

```diff
@@ -45,21 +54,20 @@ open class _ChatMessageComposerCommandCellView<ExtraData: ExtraDataTypes>: View,
     }
 
     override open func updateContent() {
-        guard let command = command else { return }
+        guard let command = content else { return }
         commandNameSubtitleLabel.text = "/\(command.name) \(command.args)"
         commandNameLabel.text = command.name.firstUppercased
 
         commandImageView.image = uiConfig.images.commandIcons[command.name.lowercased()]
             ?? uiConfig.images.messageComposerCommandFallback
     }
 
-    // MARK: Private
-
     private func setupLeftImageViewConstraints() {
         commandImageView.leadingAnchor.pin(equalTo: layoutMarginsGuide.leadingAnchor).isActive = true
         commandImageView.topAnchor.pin(equalTo: layoutMarginsGuide.topAnchor).isActive = true
         commandImageView.bottomAnchor.pin(equalTo: layoutMarginsGuide.bottomAnchor).isActive = true
         commandImageView.widthAnchor.pin(equalTo: commandImageView.heightAnchor).isActive = true
+        commandImageView.centerYAnchor.pin(equalTo: layoutMarginsGuide.centerYAnchor).isActive = true
```

Comment:
I am just curious why it's `layoutMarginsGuide.centerYAnchor` and not just `centerYAnchor` :)

Reply:
To be honest, I don't know 😅 It is probably the same thing? just noticed I used the layoutMarginsGuide, probably copy paste problem 😄

Reply:
ahh okay it doesn't matter, thanks :)